.github/workflows/runner-e2e-tests-podman.yml

```diff
@@ -75,20 +82,17 @@ jobs:
     env:
       SHA: ${{ inputs.sha || 'null' }}
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      PMM_UI_TESTS_BRANCH: ${{ inputs.pmm_ui_tests_branch || 'main' }}
-      PMM_QA_BRANCH: ${{ inputs.pmm_qa_branch || 'main' }}
-      DOCKER_VERSION: ${{ inputs.pmm_server_image || 'perconalab/pmm-server:dev-latest' }}
-      CLIENT_DOCKER_VERSION: ${{ inputs.pmm_client_image || 'perconalab/pmm-client:dev-latest' }}
-      CLIENT_IMAGE: ${{ inputs.pmm_client_image || 'perconalab/pmm-client:dev-latest' }}
-      CLIENT_VERSION: ${{ inputs.pmm_client_version || 'dev-latest' }}
+      PMM_UI_TESTS_BRANCH: ${{ inputs.pmm_ui_tests_branch || 'v3' }}
+      PMM_QA_BRANCH: ${{ inputs.pmm_qa_branch || 'v3' }}
+      QA_INTEGRATION_BRANCH: ${{ inputs.qa_integration_branch || 'v3' }}
```

Comment:
do we need new a new input param ?, qa_integration_branch:, can we not modify pmm_qa_branch ?